manual/sphinx/conf.py

```diff
@@ -31,7 +31,8 @@
 import subprocess
 import sys
 
-sys.path.append("../../tools/pylib")
+sys.path.append("../../tools/pylib/boutpp")
+sys.path.append("../../tools/pylib/boutconfig")
```

Comment:
Why did you do this?
That breaks the RTD build:
https://bout-dev--2844.org.readthedocs.build/en/2844/_apidoc/boutpp.html#boutpp-api
https://bout-dev.readthedocs.io/en/latest/_apidoc/boutpp.html#boutpp-api

The tools are directly in boutpp and boutconfig ...

Reply:
This was to stop generating the docs for the other directories under pylib, and to be explicit about what's built here. It looks like it built the docs for everything except these two, which is very odd. Not sure what's going on!

Reply:
I think the other ones get pip installed?
I haven't checked though!

Anyway, for zoidberg we currently don't build the docs but reference the bout++ docs, so if you want to drop zoidberg here, we should add zoidberg independently.
Also, it would be better to remove the sections, rather then to have them being empty because python cannot find the modules.